### ISSUE TITLE
There seems to be a bug with the PEP. Reporting bug with Gabriel E. and Ieshea Y. (Inspection Numbers)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#10888]
> What application are you using?

Parking Enterprise Portal

> Describe the problem.

There seems to be a bug with the PEP. It is not filtering Gabriel E. and Ieshea Y. on the "Inspections" section into the "Reports" tab; "Officer Metrics", "Inspection #'s.

I would also like to request to filter to the current fiscal year and only capture active staff only. 

> Website Address

PEP (Parking Enterprise Portal)

Internet Browser: Chrome

> Requested By
Shavyvonne W.

[Attachment](https://api.knack.com/v1/applications/595d00ebd315cc4cb98daff4/download/asset/63862dcce7b11b0021bae50d/completedinspectionsbyofficer.png) (122.27kb)

Request ID: DTS22-105790

**Comment by dianamartin** [comment]
Synced with Shavyvonne on 11/29, 
- I would also like to request to filter to the current fiscal year and only capture active staff only
- [x] Added source filter to show only records after 9/30/2022
- [x] Added field filter to only show "User Status" - Active
- [x] This helped with the page performance

**Comment by dianamartin** [comment]
Messaged Shavyvonne 11/29
![image](https://user-images.githubusercontent.com/21112975/204665904-bc007d6e-75e3-4f72-9205-2e5a42e7fc6f.png)
Hey Shavyvonne, I ended up populating that spreadsheet that I exported bc I had time and reviewed all the CSRs for Gabe and Ieshea for Oct & Nov. I was able to update the spreadsheet and import it into the system which updated the numbers for them. 

Let me know if this works for you.
